metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1284,7 +1284,7 @@ class MetalsLspService(
       // reliable way to await until those notifications appear.
       for {
         targetroot <- buildTargets.targetRoots(report.getTarget)
```

Comment:
Maybe we should do `toTargetRoot` inside `buildTarget.targetRoot` method? Otherwise we might forget to put it in one place